Yona/R.generated.swift

```diff
@@ -278,6 +278,10 @@ struct R {
     static var imgTime: UIImage? { if #available(iOS 8.0, *) { return UIImage(named: "imgTime", inBundle: _R.hostingBundle, compatibleWithTraitCollection: nil) } else { return UIImage(named: "imgTime") } }
     static var imgTransparantie: UIImage? { if #available(iOS 8.0, *) { return UIImage(named: "imgTransparantie", inBundle: _R.hostingBundle, compatibleWithTraitCollection: nil) } else { return UIImage(named: "imgTransparantie") } }
     static var second: UIImage? { if #available(iOS 8.0, *) { return UIImage(named: "second", inBundle: _R.hostingBundle, compatibleWithTraitCollection: nil) } else { return UIImage(named: "second") } }
+    static var slideDelen: UIImage? { if #available(iOS 8.0, *) { return UIImage(named: "slideDelen", inBundle: _R.hostingBundle, compatibleWithTraitCollection: nil) } else { return UIImage(named: "slideDelen") } }
```

Comment:
Change the variable name to the English name
Instead of:
static var slideDelen 

Write
static var sharingSlide (Delen means sharing in Dutch, like share on facebook or whatever)